src/scripts/common.ts

```diff
@@ -381,20 +381,25 @@ namespace Scripts {
         }
 
         static turboRessFull() {
+            const getBloodyBandageGraphic = () => {
+                if (Orion.Count(gameObject.uncategorized.krvavaBanda1.graphic) >= 30) {
+                    return gameObject.uncategorized.krvavaBanda1.graphic;
+                } else if (Orion.Count(gameObject.uncategorized.krvavaBanda2.graphic) >= 30) {
```

Comment:
tady bych jen dodal ze else if neni potreba kdyz mas predtim return :) ale to je jen pracovni uchylka toto komentovat :)